plugins/CoreAdminHome/Emails/SecurityNotificationEmail.php

```diff
@@ -0,0 +1,85 @@
+<?php
+/**
+ * Matomo - free/libre analytics platform
+ *
+ * @link https://matomo.org
+ * @license http://www.gnu.org/licenses/gpl-3.0.html GPL v3 or later
+ *
+ */
+
+namespace Piwik\Plugins\CoreAdminHome\Emails;
+
+use Piwik\Mail;
+use Piwik\View;
+use Piwik\Piwik;
+
+abstract class SecurityNotificationEmail extends Mail
+{
+    public static $notifyPluginList = [
+        'Login' => 'Brute Force',
+        'TwoFactorAuth' => 'Two-factor auth',
+        'CoreAdminHome' => 'CORS'
```

Comment:
I can't use the `translate` function here, because it's static. what's your recommendation @diosmosis?

Reply:
You could set the values to the translation keys:

```
'Login' => 'CoreAdminHome_BruteForce',
...
```

then call `Piwik::translate($notifyPluginList['Login'])`. It's done this way in quite a few places.

Reply:
what a great idea, I didn't think about it, thanks 😃